dask/dataframe/io/tests/test_parquet.py

```diff
@@ -676,7 +697,8 @@ def test_append_with_partition(tmpdir, engine):
     out = dd.read_parquet(
         tmp, engine=engine, index="index", gather_statistics=True
     ).compute()
-    out["lon"] = out.lon.astype("int")  # just to pass assert
+    # convert categorical to plain int just to pass assert
+    out["lon"] = out.lon.astype(df0.lon.dtype)
```

Comment:
There was a int64 vs int32 issue on windows in the 3.8 env with pandas 1.2.0 (https://github.com/dask/dask/pull/6996/checks?check_run_id=1681175945#step:5:249). Not directly sure how that would be caused by pandas, though, or why it started failing now.

Reply:
Hmm, still failing with this change ..